fakeservice/README.md

```diff
@@ -0,0 +1,184 @@
+# Fake Service
+
+Fake service to emulate how to nova and services consume oslo.messaging.
+
+The main goal of this fake sub project is to reproduce similare behaviour
+that openstack service who will use oslo.messaging rabbitmq driver.
+
+# Nova use case inspect
+
+This section would describe the nova use case and how nova
+consume oslo.messaging and the mechanismes under the hood to
+lunch the rabbitmq driver.
+
+Well, let's go!
+
+#### The puppet part
+
+Nova provide a [puppet project](https://github.com/openstack/puppet-nova/)
```

Comment:
Note that Nova doesn't provide this. puppet-nova is written by the Puppet team. Nova developers have probably contributed to it, but it's not owned by Nova.

It's also important to note that Puppet is not the only thing used to deploy Nova. The other major one I'm aware of would be Ansible, as part of the OpenStack Ansible project. There are undoubtedly many others in use in the wild too (Chef, Salt, etc.).